app/models/manageiq/providers/hawkular/middleware_manager/alert_profile_manager.rb

```diff
@@ -25,11 +26,11 @@ def update_alerts(profile_id, old_alerts_ids, new_alerts_ids, old_assignments_id
         to_remove_alerts_ids = old_alerts_ids - new_alerts_ids
         to_add_alerts_ids = new_alerts_ids - old_alerts_ids
         to_remove_alerts_ids.each do |alert_id|
-          group_trigger = @alerts_client.get_single_trigger "MiQ-#{alert_id}", true
+          group_trigger = @alerts_client.get_single_trigger(@ems.miq_id_prefix("alert-#{alert_id}"), true)
```

Comment:
Why not move the `miq_id_prefix` call to `get_single_trigger`? Either that or moving it to a different class altogether. Having it here, and specially duplicated later, generate pain points and [Divergent Changes](https://sourcemaking.com/refactoring/smells/divergent-change), making it easier to generate bugs if we change one but not the other.